docs/classes/didtools_multidid.Multidid.md

```diff
@@ -0,0 +1,161 @@
+# Class: Multidid
+
+[@didtools/multidid](../modules/didtools_multidid.md).Multidid
+
+## Constructors
+
+### constructor
+
+• **new Multidid**(`code`, `id`, `url`)
+
+#### Parameters
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `code` | `number` | DID Method Codec |
+| `id` | `Uint8Array` | DID method id |
+| `url` | `Uint8Array` | DID Method url portion |
+
+## Accessors
+
+### bytes
+
+• `get` **bytes**(): `Uint8Array`
+
+Get the multidid bytes
+
+#### Returns
+
+`Uint8Array`
+
+___
+
+### methodCode
+
+• `get` **methodCode**(): `number`
+
+Get the multidid method code
+
+#### Returns
+
+`number`
+
+___
+
+### methodIdBytes
+
+• `get` **methodIdBytes**(): `Uint8Array`
+
+Get the multidid method id bytes
```

Comment:
Is it worth exposing this in the first place as it might cause confusion?

Reply:
Do you want to drop `methodCode()`, `methodIdBytes()`, `urlBytes()` and just make the constructer take a DID string? Then code, method, and url are just internal details.

Reply:
would prefer keep constructor low level with all necessary parts, similar to cids libraries and keep parts available in some easy way, parts could also be returned from an `inspect()` function that returns json object with all bytes for each part instead of these funcs, that would likely be simpler / more clear 

but for usability everyone will still use `fromBytes` and `fromString` to create an instance